tests/validation/tests/v1_api/resource/project.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: management.cattle.io/v3
+kind: Project
+metadata:
+  name: project-2
+  namespace: c-pqtg5
+spec:
+  clusterName: c-pqtg5
```

Comment:
Is this cluster ID or cluster name? Why is this hardcoded?

Reply:
It is a placeholder and the value from where I got the template. The real value will replace it in the tests.

Reply:
the placeholder values are all empty strings now.